docs/public/presets/toml/gruvbox-rainbow.toml

```
@@ -93,8 +102,29 @@ format = '[[ $symbol $branch ](fg:color_fg0 bg:color_aqua)]($style)'
 
 [git_status]
 style = "bg:color_aqua"
+ahead = '⇡${count}'
+diverged = '⇕⇡${ahead_count}⇣${behind_count}'
+behind = '⇣${count}'
+untracked = '?${count}'
+staged = '+${count}'
+renamed = '»${count}'
+deleted = '✘${count}'
+modified = '!${count}'
```

Since this looks like the default with `$count` added I would prefer not adding it to the preset.
```suggestion
ahead = '⇡${count}'
diverged = '⇕⇡${ahead_count}⇣${behind_count}'
behind = '⇣${count}'
untracked = '?${count}'
staged = '+${count}'
renamed = '»${count}'
deleted = '✘${count}'
modified = '!${count}'
```